src/core/src/type/element_type.cpp

```diff
@@ -109,7 +105,28 @@ ov::element::Type::Type(size_t bitwidth,
                         bool is_signed,
                         bool is_quantized,
                         const std::string& /* cname */) {
-    for (const auto& t : get_type_info_map()) {
+    const ElementTypes::ElementsMap elements_map{
```

Comment:
this is quite unusual constructor which seems not covered by existing unit tests.

Reply:
@ilyachur can it be removed?

Reply:
Do you mean remove the constructor? No we cannot. But we can deprecate it and remove in the furure